### DISCUSSION TITLE
Remove underlines from headers

### DISCUSSION THREAD

**Discussion by m-schoder** [requarks/wiki#4700] [Help / Questions]
Hi there!

Is there a way to remove underlines from specific headers? (h3 downwards)

Thank you for your help!

Kind regards,
Michael

PS: I am loving your wiki. Thanks to all developers!

**Comment by ieaster1** [comment]
You can use the css code injection for this.

Admin -> Theme -> CSS Override

You could add something like:
```
h3::after, h4::after, h5::after, h6::after {
  background: none !important;
}
```

If you want to make more alterations, you should review [theme documentation](https://docs.requarks.io/dev/themes).
